observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,63 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="^1,^1,^1,^1,^1,^1", options="header"]
+|===
+| *COO Version*   | *OCP Versions*       | *Dashboards*   | *Distributed Tracing*   | *Logging*   | *Troubleshooting Panel* 
+
+| 0.2.0           | 4.11                 | ✔              | ✘                       | ✘           | ✘ 
+| 0.3.0+          | 4.11 - 4.15          | ✔              | ✔                       | ✔           | ✘ 
+| 0.3.0+          | 4.16+                | ✔              | ✔                       | ✔           | ✔ 
```

Comment:
Nitpick: AsciiDoc allows you to fuse the two cells for `0.3.0+` in the `COO Version` column into a single cell:
https://docs.asciidoctor.org/asciidoc/latest/tables/span-cells/#span-multiple-rows

Reply:
This is true, but I also hear that merging cells can make it more difficult for those using a screen reader.

Reply:
As these docs originate upstream, I am going to leave as is, otherwise the span would have to be inserted every time they are re-converted